native/exe/MiniClientLauncher/MiniClientLauncher.cpp

```diff
@@ -150,7 +150,7 @@ int APIENTRY WinMain(HINSTANCE hInstance,
 	LPTSTR appPath = new TCHAR[512];
 	GetModuleFileName(exeMod, appPath, 512);
 	size_t appLen = strlen(appPath);
-	for (size_t i = appLen - 1; i > 0; i--)
+	for (size_t i = appLen; i-- > 0;) // this does the check for continuation on the value before decrement, but still uses the decremented value inside the loop 

```

Comment:
A much cleaner solution is to just put that for loop inside an 'if (appLen > 0)' conditional block...then you don't need an ugly comment like this. :)

Reply:
Was tossing up whether to just put it inside an 'if (appLen > 0)' conditional, then I read [this](https://stackoverflow.com/questions/1951519/should-i-use-stdsize-t-or-int-in-my-for-loops)  :)
Changed to  'if (appLen > 0)' conditional

Reply:
Looks like you missed this one...

Reply:
And that's why there needs to be at least another pair of eyes looking at it :)
Done. Thanks